src/dmd/mtype.d

```diff
@@ -5770,6 +5784,11 @@ extern (C++) final class TypeEnum : Type
         return memType().isintegral();
     }
 
+    override bool ischaracter()
```

Comment:
This creates a virtual function, but the virtual-ity is never used in anywhere. I also suspect the 3 tests are far more efficient than virtual dispatch.